tokio/tests/sync_mpsc.rs

```diff
@@ -418,9 +418,33 @@ fn dropping_rx_closes_channel_for_try() {
         }
     }
 
+    {
+        let err = assert_err!(tx.try_reserve());
+        match err {
+            TrySendError::Closed(..) => {}
+            _ => panic!(),
+        }
+    }
+
+    {
+        let err = assert_err!(tx.try_reserve_owned());
+        match err {
+            TrySendError::Closed(..) => {}
+            _ => panic!(),
+        }
+    }
```

Comment:
```suggestion
    assert!(matches(tx.try_reserve_owned(), Err(TrySendError::Closed(_))));
```

Reply:
I only did them this way to be consistent with the other assertion in this function without modifying the existing assertion. Should I just reformat all 3?

Reply:
OK, I just reformatted all 3, at which point they became an exact duplicate of the `try_reserve_on_closed_channel()` test I added, so I've removed that too.

Also remembered to run the formatter this time :facepalm: